release-notes/opensearch-release-notes-2.0.0.md

```diff
@@ -0,0 +1,637 @@
+* Document level security now supports term lookup queries.
+
+### OpenSearch Notifications
+* OpenSearch 2.0.0 is the first official release with OpenSearch Notifications.
+* Notifications consist of three plugins, `notifications-core` and `notifications` backend plugins for OpenSearch, and a `notificationsDashboards` frontend plugin for OpenSearch Dashboards.
+
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.0.0 include the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.0/release-notes/opensearch.release-notes-2.0.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.0/release-notes/opensearch-dashboards.release-notes-2.0.0.md).
+
+## FEATURES
+
+### OpenSearch Index Management
+* Adds shrink action to ISM ([#326](https://github.com/opensearch-project/index-management/pull/326))
+* Notification integration with IM ([#338](https://github.com/opensearch-project/index-management/pull/338))
+
+
+### OpenSearch Index Management Dashboards Plugin
+* Adds UI for shrink action ([#176](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/176))
+
+
+### OpenSearch Job Scheduler
+* Enables locking on an arbitrary lockID ([164](https://github.com/opensearch-project/job-scheduler/pull/164))
+
+
+### OpenSearch Notifications
+* Use Notifications to manage your notification channel configurations in a centralized location and send messages to these channels.
+* This release also adds notification-specific REST APIs for CRUD operations on channels and internal transport APIs for integrating/communicating with other plugins.
+
+
+### OpenSearch Performance Analyzer
+* Adds setting to enable/disable Thread Contention Monitoring ([#171](https://github.com/opensearch-project/performance-analyzer/pull/171))
+* Fixes calculation of average thread blocked time and average thread waited time ([#118](https://github.com/opensearch-project/performance-analyzer-rca/pull/118))
+
+
+### OpenSearch SQL
+* Add Kmeans and AD command documentation ([#493](https://github.com/opensearch-project/sql/pull/493))
+* AD and Kmeans grammar edits ([#500](https://github.com/opensearch-project/sql/pull/500))
+* Support more parameters for AD and KMEANS command, and update related documentation ([#515](https://github.com/opensearch-project/sql/pull/515))
+
+
+## ENHANCEMENT
+
+### OpenSearch Alerting
+* Add automated migration for Destinations to Notification configs ([#379](https://github.com/opensearch-project/alerting/pull/379]))
+* Integrate with Notifications plugin for Alerting backend ([#401](https://github.com/opensearch-project/alerting/pull/401]))
+* Integrate Document Level Alerting changes ([#410](https://github.com/opensearch-project/alerting/pull/410]))
```

Comment:
Document level alerting should be under the feature section since it is a net new feature in alerting.